guides/common/modules/proc_viewing-image-mode-hosts-and-booted-container-images.adoc

```diff
@@ -0,0 +1,26 @@
+[id="Viewing_image_mode_hosts_and_booted_container_images_{context}"]
+= Viewing image mode hosts and booted container images
+
+These actions streamline the management of image mode hosts, providing flexibility and control over host operations through {Project}.
+
+.Prerequisites
+* *Pull-Mode Remote Execution*: For pull-mode remote execution users, related packages must be included in the image mode host's container image.
+
+.Procedure
+. On the new Hosts Overview page, navigate to *Content* > *Booted Container Images* to view all images used by image mode hosts.
+This page displays the spread of digests under specific image paths.
+More digests under an image path indicate greater drift in the host ecosystem, ideally minimized by ensuring all hosts run the most up-to-date image version.
+. Click the host count to navigate to the *All Hosts Listing* page, which identifies the hosts associated with each specific image and displays the number of hosts using each digest.
+. On a host's *Details* tab, use the image mode *Details* card to see the current `bootc` status.
+This card includes a link to the *Bootc Action - Script Default* remote execution job.
+. Use the new remote execution jobs under the Bootc job category to manage image mode hosts:
+* *Bootc Action* - Script Default: Select any Bootc action from the dropdown menu, enter arguments, and specify a target.
+* *Bootc Rollback* - Script Default: Run Bootc rollback to revert to a previous image.
+* *Bootc Status* - Script Default: Run Bootc status with optional extra parameters to check the status.
+* *Bootc Switch* - Script Default: Run Bootc switch with a mandatory image input and optional extra parameters to switch to a different image.
+* *Bootc Upgrade* - Script Default: Run Bootc upgrade with optional extra parameters to update the image.
```

Comment:
Should there be a procedure for each of those actions? They aren't for viewing image mode hosts as specified by the title.

Reply:
Actually, I think that we might need to split this info into:

1. Procedure for Viewing booted container images.
2. Reference for the REX jobs, eg. Managing image mode hosts.

Reply:
Done.